test/plots/piePlotTests.ts

```diff
@@ -463,77 +427,53 @@ describe("Plots", () => {
         assert.strictEqual(entity3.length, 0, "no entities returned");
```

Comment:
This is a duplicate message. Let's have unique assert messages. Maybe `no entities returned for click 3`